launchScripts/genesisWellKnownDidConfig.ts

```diff
@@ -204,7 +204,7 @@ async function main() {
   await Kilt.disconnect()
 
   console.log(
-    'this is the well-known-DID-configuration of your dApp that was just created: \n',
+    "this is your dApp's Domain Linkage Credential (well-known-DID-configuration) that was just created: \n",
```

Comment:
This shouldn't be part of the changes, we can make a separate ticket for this

Reply:
Move it to #84